api/db-cdr/csv/domain_vocabulary_info.csv

```diff
@@ -0,0 +1,36 @@
+"domain_id","vocabulary_id","all_concept_count","standard_concept_count"

```

Comment:
Hmm.. Doesn't there need to be some script or process for regenerating this file? I assume this exists for the other CSVs here.

Reply:
There is a script for generating this data when producing data in Cloud SQL. However, for the purposes of setting up a database locally, we have CSV files in github that we load in directly. See other CSVs in this dir.

This data originally was pulled from Cloud SQL. We don't at present have scripts for grabbing this CSV data out -- I did it manually -- but that's not super-important; we don't really need to make sure these counts for local development match CDR versions we use in production. (And if we were going to try to script it, that would be a different PR.)

Reply:
Thanks, I'm confused every time I look at these CSV files - filed https://precisionmedicineinitiative.atlassian.net/browse/RW-1493 to clarify the roles.